src/SPECS/hootenanny.spec.in

```diff
@@ -196,7 +200,10 @@ if [ "$1" = "1" ]; then
     fi
 
     # restore saved db config file if present
-    if [ -f /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave ]; then
+
+    if [ -f /var/lib/hootenanny/conf/DatabaseConfigDefault.sh.rpmsave ]; then
+        sudo mv /var/lib/hootenanny/conf/DatabaseConfigDefault.sh.rpmsave /var/lib/hootenanny/conf/DatabaseConfigDefault.sh
+    elif [ -f /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave ]; then
         sudo mv /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave /var/lib/hootenanny/conf/DatabaseConfig.sh
```

Comment:
Assuming DatabaseConfigLocal.sh is an untracked file that gets ignored by the rpm, I'm not sure we need any of this code.  Maybe a onetime switch to copy the DB_PASSWORD line from DatabaseConfig.sh.rpmsave to DatabaseConfigLocal.sh if this file does not exist.